### ISSUE TITLE
Cannot use + or append() for string operations in naming parameters

### ISSUE THREAD

**Issue by tdhung80** [orientechnologies/orientdb#5572]
Hi,

I am migrating from version 1.7.8 to version 2.1.8, I get an error with such query in version 2.1.8

```SQL
select from User where Account.Name like :name + '%'
```

it's also getting same error with
```SQL
select from User where Account.Name like :name.append('%')
```
it's only working with this
```SQL
select from User where Account.Name like :name
```
Remarks, all style queries are working if strictSql is unset, what's the correct syntax in this case when strictSQL is set?

My Best,
Hung Tran

**Comment by luigidellaquila** [comment]
Hi @tdhung80 

Thanks, I'll check it asap

Luigi

**Comment by luigidellaquila** [comment]
Hi @tdhung80 

This issue is fixed in 2.2 GA, the right syntax is the first one:

```sql
select from User where Account.Name like :name + '%'
```

Thanks

Luigi
